core/storage/suggestion/gae_models.py

```diff
@@ -72,6 +74,12 @@
 # The delimiter to be used in score category field.
 SCORE_CATEGORY_DELIMITER = '.'
 
+# Threshold time after which suggestion is considered stale and auto-accepted.
+THRESHOLD_TIME_BEFORE_ACCEPT = 7 * 24 * 60 * 60 * 1000
+
+# The default message to be shown when accepting stale suggestions.
+DEFAULT_SUGGESTION_ACCEPT_MESSAGE = 'Accepting suggestion due to inactivity.'
```

Comment:
Message is not clear -- whose inactivity?

"Automatically accepting suggestion after X days." would be better. Make sure X is a named constant which gets interpolated and that it is also used in THRESHOLD_TIME_BEFORE_ACCEPT.

Reply:
Done